apps/tx-builder/src/components/forms/fields/SelectContractField.tsx

```diff
@@ -18,6 +18,10 @@ const SelectContractField = ({
 }: SelectContractFieldTypes) => {
   return (
     <Select
+      id={name}
```

Comment:
is it guaranteed that the name wil be unique across all elements of the page? if not I'd use testId instead

Reply:
good catch!

now I am using the `id` prop from `Field` component instead of the `name`. because `name` prop
might not be unique across all elements of the page.

```
const SelectContractField = ({
  value,
  onChange,
  options,
  label,
  name,
  id,
}: SelectContractFieldTypes) => {
  return (
    <Select
      id={id}
      inputProps={{
        id: `${id}-input`,
      }}
      name={name}
```